.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# GLOBAL OWNER
+*   @ProjectPythia/infrastructure
```

Comment:
I don't think this will work.  You can't assign ownership to a team in another GitHub organization.  You need to create a new team in the ProjectPythiaTutorials organization.

Reply:
Made a new team!